flink-formats/flink-hadoop-bulk/src/test/java/org/apache/flink/formats/hadoop/bulk/HadoopPathBasedPartFileWriterTest.java

```diff
@@ -70,6 +70,7 @@ public void testPendingFileRecoverableSerializer() throws IOException {
 
     @Test
     public void testWriteFile() throws Exception {
+        // create a tmp folder with random name using jdk file api
```

Comment:
I'm curious what the intention of this comment is: Doesn't it just describes what the next line does? It feels a bit obsolete. WDYT?

Reply:
Remove the commet